dts/arm/ti/j721e-pinctrl.dtsi

```diff
@@ -0,0 +1,25 @@
+/* Copyright (C) 2023 BeagleBoard.org Foundation
+ * Copyright (C) 2023 S Prashanth
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr/dt-bindings/pinctrl/ti-k3-pinctrl.h>
+
+&pinctrl {
+	uart0_tx_default: uart0_tx_default {
+		pinmux = <0x1e8 (PIN_OUTPUT | 0)>;
+	};
+
+	uart0_rx_default: uart0_rx_default {
+		pinmux = <0x1ec (PIN_INPUT | 0)>;
+	};
+
+	uart2_tx_default: uart2_tx_default {
+		pinmux = <0x1c (PIN_OUTPUT | 14)>;
```

Comment:
Why is it 14? What pin is 0x1c? Add comments please.

Reply:
Or else make it as some #defines, as long as a person reading the code doesn't have to dig around TRMs / datasheets to just understand what 14 and 0 correspond to